LoopBack/LBRESTAdapter.m

```diff
@@ -34,6 +34,23 @@ - (void)setAccessToken:(NSString *)accessToken {
     [self saveAccessToken:accessToken];
 }
 
+- (LBModelRepository *)repositoryWithModelName:(NSString *)name {
+    NSParameterAssert(name);
+
+    LBModelRepository *repository = [LBModelRepository repositoryWithClassName:name];
+    [self attachRepository:repository];
+    return repository;
+}
+
+- (LBPersistedModelRepository *)repositoryWithPersistedModelName:(NSString *)name {
+    NSParameterAssert(name);
+
+    LBPersistedModelRepository *repository = [LBPersistedModelRepository repositoryWithClassName:name];
+    [self attachRepository:repository];
+    return repository;
+}
+
+// The following method has been deprecated
```

Comment:
@hideya Would it make sense to rewrite the implementation of this deprecated method to use the new methods?

```
{
  if (persisted)
    return [adapter repositoryWithPersistedModelName:name];
  else
    return [adapter repositoryWithModelName:name];
}
```